django_mongodb_backend/indexes.py

```diff
@@ -101,6 +105,185 @@ def where_node_idx(self, compiler, connection):
     return mql
 
 
+class SearchIndex(Index):
+    suffix = "six"
+    _error_id_prefix = "django_mongodb_backend.indexes.SearchIndex"
+
+    def check(self, model, connection):
+        errors = []
+        if not connection.features.supports_atlas_search:
+            errors.append(
+                Warning(
+                    "This MongoDB server does not support atlas search.",
+                    hint=(
+                        "The index won't be created. Silence this warning if you "
```

Comment:
How about amending the second sentence: "Use an Atlas-enabled version of MongoDB, or silence..."